src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,83 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number
+  currentPage: number,
+  onPageChange: (newPage: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageCount = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pageCount);
+  const nextPage = currentPage + 1;
+  const prevPage = currentPage - 1;
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', {
+        disabled: prevPage < 1,
```

Comment:
move to the variable & reuse in all cases
```suggestion
        disabled: ...,
```